scripts/zones/Chateau_dOraguille/npcs/_6h0.lua

```diff
@@ -59,7 +59,7 @@ entity.onTrigger = function(player, npc)
         player:startEvent(90) -- Start
 
     -- Trust: San d'Oria (Trion)
-    elseif player:getRank() == 6 and player:hasKeyItem(tpz.ki.SAN_DORIA_TRUST_PERMIT) and not player:hasSpell(905) then
+    elseif player:getRank() >= 6 and player:hasKeyItem(tpz.ki.SAN_DORIA_TRUST_PERMIT) and not player:hasSpell(905) then
```

Comment:
Adding additional references, I think this needs to be >= (still definitely an issue, since if you pass Rank 6 without claiming, it wouldn't work.

https://www.bg-wiki.com/ffxi/BGWiki:Trusts#Trion
https://ffxiclopedia.fandom.com/wiki/Trust:_Trion